src/hyperion/system_tests/test_main_system.py

```diff
@@ -117,8 +121,9 @@ def mock_dict_values(d: dict):
 
 
 @pytest.fixture
-def test_env():
+def test_env(request):
     mock_run_engine = MockRunEngine()
```

Comment:
Nit: Better if the `MockRunEngine` requires a test name on init, then we don't accidentally end up with logs saying the test was called `test`